src/shared/components/mutationTable/column/AnnotationColumnFormatter.tsx

```diff
@@ -185,6 +224,12 @@ export default class AnnotationColumnFormatter
                         is3dHotspot={annotation.is3dHotspot}
                     />
                 </If>
+                <If condition={columnProps.enableCivic || false}>
+                    <Civic
+                        civicEntry={annotation.civicEntry}
+                        isCivicDisabled={annotation.isCivicDisabled}
```

Comment:
Again, this name is confusing. The disabled icon only works on CNA when you have information about the gene but not about CNA variants, so when you hover over the gene you see the description of the gene.